assets/src/edit-story/components/canvas/utils/normalizeRotationDegrees.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+function normalizeRotationDegrees(degrees) {
+  return ((degrees % 360) + 360) % 360;
```

Comment:
I know it's been like this before, but still wondering why not
```suggestion
 */
 
function normalizeRotationDegrees(degrees) {
  return degrees % 360;
```

Reply:
Ping @cvolzke4 since this was originally implemented in https://github.com/google/web-stories-wp/pull/1457 -- was there a reason for not just using `degrees % 360`?

Reply:
If we want both `540`, `-180` and `-540` to be converted to `180`, then we need this entire line of calculation as modulo returns negative numbers too.

It's pretty standard for _"positive-only modulo"_, so I don't think it'll cause too much confusion, but it could be amended with a short comment.

Reply:
Thanks for the explanation.
> as modulo returns negative numbers too

TIL. Found this long list of languages and how they handle it
![image](https://user-images.githubusercontent.com/21036224/91226145-ee3c7580-e724-11ea-8a4e-816113ff4afd.png)
and [The JavaScript Modulo "Bug" ](https://web.archive.org/web/20090717035140if_/javascript.about.com/od/problemsolving/a/modulobug.htm)

Reply:
Just noting that it's not really required (and we would be fine with negative numbers - we have them in the input and CSS rotate accepts them).

Reply:
Added an additional comment for clarity.